dev-scripts/prepare_package_xml.sh

```diff
@@ -133,12 +136,17 @@ do
     echo '</dir>'
     continue
   fi
+  # skip the src directory as it is not part of the native extension
+  if [[ ${a} == 'src' ]]
+  then
+    continue
+  fi
+
   if [[ -f ${a} ]]
   then process_file "${a}"
   else process_dir "${a}"
   fi
 done
 echo '</dir>'
 
-cat package.xml-template_post
-
+cat $TOOLS_DIR/package.xml-template_post
```

Comment:
since we're moving to publishing this repo into 2 different package managers, should we start naming files like "package-pecl.xyz" and "package-composer.xyz" if they're specific to one of the package managers?

I know that would help me as a newcomer follow what's going on, vs a file just being named "package.xyz" and you just kinda gotta know which package manager uses this particular file

Reply:
Well, but those package has already been released for a long while. Specially the one in composer, https://packagist.org/packages/aws/aws-crt-php, has been installed 60 238 167 times.

I don't think it's a good idea to change them now. And, it's two different things in two different package manager, I think it's not too bad.

Reply:
Oh, sorry, I misread your comment.

Yeah, I renamed the helper script to have pecl in it. Totally agree.